hooks/pre-commit

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env ruby
```

Comment:
Happy to move this to a separate commit if it makes reviewing easier! Would be easy to do.